### PR TITLE
Supply shuttle no longer sends with 0 orders

### PR DESCRIPTION
**Pull request by Kurfursten**
Gives error message instead

Fully tested

### CODE REVIEW

**Comment by BroderTuck** [comment]
Yes! Great work! You are number 1!

**Comment by ghost** [comment]
Can you just do a y/n? It's nice to gib cuffed people with a shuttle and needing sheckles to do so after you've already blown all your dosh on spec ops crates is a bit ow.

**Comment by Probe1** [comment]
Just add a confirm.  I often send things as an admin using it.  It's a nice IC way of giving the players something to play with.

File: code/game/supplyshuttle.dm [hunk 1]
```diff
@@ -611,6 +611,7 @@ var/list/mechtoys = list(
 			temp = "You aren't able to establish contact with central command, so the shuttle won't move. <BR><BR><A href='?src=\ref[src];mainmenu=1'>OK</A>"
 		else if(!supply_shuttle.can_move())
 			temp = "For safety reasons the automated supply shuttle cannot transport live organisms, classified nuclear weaponry or homing beacons.<BR><BR><A href='?src=\ref[src];mainmenu=1'>OK</A>"
+		else if(!supply_shuttle.shoppinglist.len) temp = "To conserve fuel, central command will not authorize a supply shuttle with zero orders.<BR><BR><A href='?src=\ref[src];mainmenu=1'>OK</A>"
```
Comment (ComicIronic) [inline code/game/supplyshuttle.dm:614]: No one-line ifs please

Comment (Kurfursten) [inline code/game/supplyshuttle.dm:614]: That's silly, it's purely stylistic. Also I have made at least one hundred PRs with one-line ifs and you've never said anything about it before.

Comment (ComicIronic) [inline code/game/supplyshuttle.dm:614]: Maybe you've made them when I wasn't around, but coding standards CAN be purely stylistic, and this is one.

Format your code properly.

Comment (Kurfursten) [inline code/game/supplyshuttle.dm:614]: No, you've merged PRs from me with one line ifs. You're being needlessly confrontational.

Comment (PJB3005) [inline code/game/supplyshuttle.dm:614]: Past != present.

**Comment by Kurfursten** [comment]
I'm not going to be party to Contributor dick waving. If you need to place meme tags and complain, that's something you need to work out without me. I don't mind coding it to add a confirm, but I don't need this garbage.

I like that your response to my PR was to immediately invent new coding standards and begin writing them with "no one line ifs" in the IRC.

**Comment by Iamgoofball** [comment]
no one line ifs is a coding standard and practice at literally every codebase and has been enforced everywhere including here

all you had to do was hit the enter key and then the tab key a few times

**Comment by ghost** [comment]
Oh and another thing, once wages are back in place or we go back to the points system you may occasionally just want the shuttle back to load empties to send back for dosh in the cargo account or supply points.

**Comment by Kurfursten** [comment]
@SarahJohnson489 I wasn't really opposed to coding it to add a confirm, it's all of 15 extra keystrokes

send=1,override=1 plus an extra if

**Comment by Iamgoofball** [comment]
do you see the irony in this

**Pull request closed without merging**